.github/styles/vocab.txt

```diff
@@ -80,6 +82,7 @@ inlinehilite
 interop
 javascript
 Javascript
+jenkins
```

Comment:
Btw, this might lead to users writing jenkins instead of Jenkins. I know why added it, as the headings contain the package name. vale might skip that if you write `jenkins` instead (in code block)

Reply:
Yeah think it looks alright with codeblocks, good idea!